src/Layout/SidebarLayout/SidebarLayout.component.tsx

```diff
@@ -0,0 +1,121 @@
+// REACT
+import * as React from 'react';
+// VENDOR
+import styled from 'styled-components';
+import { Grid, Cell } from 'styled-css-grid';
+import classnames from 'classnames';
+// COMPONENTS
+import { CustomAdaptor, StandardBreakpoints } from '../../Grid/Adaptor/index';
+
+// Constants
+// ------------------------------------------------------------------------------------------------------------------
+export const DEFAULT_LAYOUT_WIDTH = '100%';
+export const DEFAULT_CONTENT_WIDTH = '71.25rem';
+export const DEFAULT_SIDEBAR_WIDTH = '13.75rem';
+export const TRANSPARENT = 'transparent';
+export const RIGHT = 'right';
+export const LEFT = 'left';
+
+// Types
+// ------------------------------------------------------------------------------------------------------------------
+type SidebarAlignment = 'left' | 'right';
+
+// Styled Components
+// ------------------------------------------------------------------------------------------------------------------
+const StyledLayout = styled.section<StyledLayoutProps>`
+    max-width: ${props => props.layoutWidth};
+    background-color: ${props => props.layoutBackgroundColor};
+    margin: 0 auto;
+`;
+
+const StyledContent = styled.div<StyledContentProps>`
+    max-width: ${props => props.contentWidth};
+    background-color: ${props => props.contentBackgroundColor};
+    margin: 0 auto;
+`;
+
+StyledLayout.displayName = 'StyledLayout';
+StyledContent.displayName = 'StyledContent';
+
+// Interfaces
+// ------------------------------------------------------------------------------------------------------------------
+interface SidebarLayoutProps {
+    /** The width of the layout, edge to edge. */
+    layoutWidth?: string;
+    /** The width of the content within the layout. */
+    contentWidth?: string;
+    /** The width of the area for the sidebar. */
+    sidebarWidth?: string;
+    /** If the sidebar is on the left or the right of the layout */
+    sidebarAlign?: SidebarAlignment;
+    /** The actual sidebar. This can be a component. */
+    sidebar?: any;
+    /** The background color of the layout. Default is transparent. */
+    layoutBackgroundColor?: string;
+    /** The background color of the content within the layout area. Default is transparent. */
+    contentBackgroundColor?: string;
+    /** Additional classname. */
+    className?: string;
+    children?: any;
+}
+
+interface StyledLayoutProps {
+    layoutWidth?: string;
+    layoutBackgroundColor?: string;
+}
+
+interface StyledContentProps {
+    contentWidth?: string;
+    contentBackgroundColor?: string;
+}
+
+// Components
+// ------------------------------------------------------------------------------------------------------------------
+export const SidebarLayout = ({
```

Comment:
Finest possible hair being split here: Do we need the added `Layout` for this component name? It sounds redundant, but I will definitely not die on this hill.

Reply:
I think w/o the word 'Layout' the usage of the component could be confusing. Dropping `<SidebarLayout>` in jsx is much more clear than `Sidebar` IMO